src/views/auth/register/register-step-one.tsx

```diff
@@ -7,13 +7,25 @@ import { IRegistrationPageStep } from './types';
 import { Icon } from '@rneui/themed';
 import { THEME_CONFIG } from '../../../config/appConfig';
 import { RegisterFormContext } from '.';
-import { Formik } from 'formik';
+import { Field, Formik } from 'formik';
 import { IRegisterPayload } from '../../../store/types';
 import { RegisterSchema_1 } from '../../../utils/schemas';
+import PhoneNumberInput from '../../../components/atoms/phone-input';
+import { isValidPhoneNumber } from 'libphonenumber-js';
 
 const RegisterStepOne: React.FC<IRegistrationPageStep> = ({ onStepPress }) => {
     const onSubmit = () => {};
 
+    const validate = (values: IRegisterPayload) => {
+        const errors: Partial<IRegisterPayload> = {};
```

Comment:
Do this instead `Pick<IRegisterPayload, 'phoneNumber'>`

Reply:
this won't work because of the empty object @chukwuma-azubuike